File: android-app/src/main/java/fr/cph/chicago/entity/dto/FirstLoadDTO.java
package fr.cph.chicago.entity.dto;

import java.util.List;

import fr.cph.chicago.entity.BikeStation;
import fr.cph.chicago.entity.BusRoute;
import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class FirstLoadDTO {
    private boolean busRoutesError;
    private boolean bikeStationsError;
    private List<BusRoute> busRoutes;
    private List<BikeStation> bikeStations;
}

File: android-app/src/main/java/fr/cph/chicago/entity/dto/FirstLoadDTO.kt
package fr.cph.chicago.entity.dto

import fr.cph.chicago.entity.BikeStation
import fr.cph.chicago.entity.BusRoute

class FirstLoadDTO(
    val busRoutesError: Boolean = false,
    val bikeStationsError: Boolean = false,
    val busRoutes: List<BusRoute>? = null,
    val bikeStations: List<BikeStation>? = null)
